modules/rke2/master/instances_server.tf

```diff
@@ -64,6 +64,97 @@ resource "aws_instance" "master" {
   }
 }
 
+
```

Comment:
on here also i would ask help/opinion to @matttrach to see if its a better solution instead creating resources separately for elastic ip = true or = false.

like , if we can work on a single "resource creation" checking this var, if its true we create the elastic, and if it false we create the "normal ones"

Reply:
Can you elaborate more on this?
If I understand correctly, this is how it is right now.
if the elastic ip = true we are going to proceed to create the elasticip and associate them to the cluster.
if the var elastic ip = false, the process is not going to execute any of the steps relate to the creation / association of the elasticip.

If I run without 

![image](https://github.com/rancher/distros-test-framework/assets/106194120/3b5040ae-9fec-457c-85e7-e49e65de0633)
 
No elastic_ip are created:
![image](https://github.com/rancher/distros-test-framework/assets/106194120/1a965834-723f-4c24-a076-d76c6ca078c2)

Setting Var elastic ip = true: 

![image](https://github.com/rancher/distros-test-framework/assets/106194120/1ed24091-6269-46a5-be88-6e3fc81f7b11)
![image](https://github.com/rancher/distros-test-framework/assets/106194120/58b4b3d5-1edd-48ec-9970-1c33531ec1bd)